packages/bvaughn-architecture-demo/components/console/renderers/shared.module.css

```diff
@@ -172,3 +195,27 @@
 .ErrorBoundaryFallback {
   color: var(--color-error);
 }
+
+:global(.theme-light) .ColorBadge {
+  --pulse-color-1: rgba(127, 127, 127, 0.1);
+  --pulse-color-2: rgba(97, 97, 97, 0);
+  --pulse-color-3: rgba(88, 88, 88, 0);
+}
+
+:global(.theme-dark) .ColorBadge {
+  --pulse-color-1: rgba(185, 185, 185, 0.35);
+  --pulse-color-2: rgba(204, 169, 44, 0);
+  --pulse-color-3: rgba(204, 169, 44, 0);
+}
```

Comment:
Any colors (including shadows) should be a CSS variable [defined here](https://github.com/replayio/devtools/blob/main/packages/bvaughn-architecture-demo/pages/variables.css) so we can easily tweak it for each theme (and so we don't end up with a lot of near-neighbor values spread around).